### DISCUSSION TITLE
Clockwork V2 Beta - April 30, 2023

### DISCUSSION THREAD

**Discussion by ToasterPanic** [red-stone-network/clockwork#45] [Announcements]
Hello everyone! Today we finished beta9, and with it comes some cool new features!

1. You can now put Clockwork apps through a proxy!
![fartbutt](https://user-images.githubusercontent.com/91223726/235355244-048d4e3c-ae05-4d07-8bc6-a7f8366286b2.PNG)

2. There's now a clear all notifications button!

3. A new menu where you can manage themes!
![fartbutt](https://user-images.githubusercontent.com/91223726/235355324-17d2a2df-16cf-4d08-8859-84265b948e06.PNG)

I'm going to have to do state testing this week, so expect development to be slowed.

Thanks for reading!

**Comment by TonyKillam** [comment]
wdym by Clockwork apps through a proxy? can i put a link in so that uv changes it so it doesnt get blocked by securly
